CRM/CiviAwards/Form/AwardReview.php

```diff
@@ -321,6 +321,7 @@ private function displayReviewForm(array $fields) {
         if (in_array($element->_attributes['name'], $elementNames)) {
           $element->freeze();
         }
+        $this->disableField($element);
```

Comment:
Any reason why don't we move this inside the `if` statement above ? We should only disable elements that are present in the `$elementNames` array I think.

Reply:
well some elements does not have name field in attributes so that is why placed it outside the if and tbh there are only some extra hidden fields and I think it wont cause any issue if I disable them in view  mode. Let me know your thoughts on it

Reply:
@shahrukh-compucorp , it's okay but the logic needs to be consistent. If we are freezing only some elements and disabling some, then there is inconsistency. It's either we disable and freeze all and remove the IF statement or we only disable and freeze those in the IF statement.

That said will suggest we move the disable function into the IF statement, since we are sure that those are the elements we want to freeze/disable.